app/views/outcomes-report/PageOutcomesReport.vue

```diff
@@ -118,8 +122,12 @@ export default {
       }
     },
     includeOther (newVal, lastVal) {
-      if (newVal !== lastVal) {
+      if (newVal) {
         this.addParametersToLocation()
+        this.loadRequiredData()
+        this.showOther = true // show other product by default
```

Comment:
lets put default as false at the time of launch, we can enable it after some testing

Reply:
nono, this true is when ozaria data loaded, let's default show it.